src/storage/tde.c

```diff
@@ -65,6 +65,11 @@
   } while (0)
 #define restore_signals(old_mask) sigprocmask(SIG_SETMASK, &(old_mask), NULL)
 
+#if defined(CS_MODE)
+DBLINK_CHPHER_KEY dblink_Cipher_key = { false, {0x00,}
+};
```

Comment:
GNU indent broke indentation
```suggestion
// *INDENT-OFF*
DBLINK_CHPHER_KEY dblink_Cipher_key = {false, {0x00,}};
// *INDENT-ON*
```

Reply:
OK, I will fix it.